aws-rotate-keys/orb.yml

```
@@ -54,7 +54,7 @@ commands:
               aws iam list-access-keys \
                 --user-name << parameters.aws-username >> \
                 --query 'AccessKeyMetadata[0].AccessKeyId' \
-                --output text 2>/dev/null && break
+                --output text >/dev/null 2>/dev/null && break
```

This can be simplified to the following:

```suggestion
                --output text 2>&1 > /dev/null && break
```